go/ethclient/mgmtcontractlib/input_handler_test.go

```diff
@@ -30,7 +30,7 @@ func TestCompression(t *testing.T) {
 		t.Fatal(err)
 	}
 	serializedRollup := EncodeToString(compressedRollup)
-	deserializedRollup := DecodeFromString(serializedRollup)
+	deserializedRollup, _ := DecodeFromString(serializedRollup)
```

Comment:
here

Reply:
I think it's fine to ignore as it's in a test.

Reply:
👍